### PR TITLE
[ZH] Update `Ranking_Criteria`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- Changes from other PRs are properly applied:
  - [x] https://github.com/ppy/osu-wiki/pull/6872
  - [x] https://github.com/ppy/osu-wiki/pull/6840
  - [x] IF/WHEN it's merged: https://github.com/ppy/osu-wiki/pull/6890

### CODE REVIEW

File: wiki/Ranking_Criteria/zh.md
Comment (RushFTK) [inline wiki/Ranking_Criteria/zh.md:52]:
```suggestion
  - **最高难度以下的难度可以使用包含休息时间的总游戏时长来替代掉血时间，作为度量标准而达到规定阈值。** 这不包含掉血时间在 30 秒以下的难度。一般由该游戏模式的 [谱面提名者](/wiki/People/The_Team/Beatmap_Nominators) (以下简称BN) 和 [提名评估团队](/wiki/People/The_Team/Nomination_Assessment_Team) (以下简称NAT) 成员来判断谱面中作为最低难度的 Hard 或者 Insane 谱面是否合格。
```

首次出现的专有名词缩写，建议保留原有全称。如果认为原有英文名称不恰当，可使用中文翻译表示。不过由于不常见，所以并不推荐。

**Review by RushFTK — DISMISSED** [review]
all looks fine for me (as a chinese native speaker). while a few point may discussion.

File: wiki/Ranking_Criteria/zh.md
Comment (RushFTK) [reply wiki/Ranking_Criteria/zh.md:52]: 这段我看了确实只有这一处出现，所以也许你不修改也可以。

Comment (YumeMuzi) [reply wiki/Ranking_Criteria/zh.md:52]: 因为中文社区内很少用全称，而且在目前没有专用的名词解释下，用缩写也可以
Abbreviations are often used to refer to them in the osu! Chinese community.

**Comment by TicClick** [comment]
I've just submitted for review https://github.com/ppy/osu-wiki/pull/6870 with small improvements for the original article -- please make sure to apply changes from it **once it's finalized and merged**

**Review by RushFTK — CHANGES_REQUESTED** [review]
Others looks fine, but keep some terms like `tags` as it was or translate should be determined.
现在这个版本的翻译中，"标签"到底是主要使用中文，在首次使用中注释英文；或者主要使用英文，在首次使用中注释中文。

File: wiki/Ranking_Criteria/zh.md
Comment (YumeMuzi) [reply wiki/Ranking_Criteria/zh.md:139]: 用英文 Tags 吧 等会改

File: wiki/Ranking_Criteria/zh.md
Comment (YumeMuzi) [reply wiki/Ranking_Criteria/zh.md:166]: #6870 已经得出结果，等会改

**Review by RushFTK — APPROVED** [review]
looks fine for a Chinese native speaker.

**Comment by Walavouchey** [comment]
let's not balloon the scope too much for this pr. https://github.com/ppy/osu-wiki/pull/6873 may take a bit to get merged

File: wiki/Ranking_Criteria/zh.md
Comment (Walavouchey) [inline wiki/Ranking_Criteria/zh.md:20]: excess line

```suggestion
```

**Comment by Walavouchey** [comment]
@RushFTK sorry, mind taking one more look for latest change ([a055904](https://github.com/ppy/osu-wiki/pull/6848/commits/a055904996fb31808b1c9f695a21b848b0e0d600))?

**Comment by TicClick** [comment]
fair, I didn't follow it closely

File: wiki/Ranking_Criteria/zh.md
Comment (RushFTK) [inline wiki/Ranking_Criteria/zh.md:13]:
````suggestion
非常建议您使用 [Mapset Verifier (MV)](https://github.com/Naxesss/MapsetVerifier) 来帮助检查谱面是否符合本文所述的规则。传统客户端 (stable 版) 内的 [AiMod](/wiki/Client/Beatmap_editor/AiMod) 工具已经过时且不再维护。同时，新版客户端( lazer 版) 内的谱面检查工具仍在[开发中](https://github.com/ppy/osu/issues/12091#issuecomment-878760791)。
````

File: wiki/Ranking_Criteria/zh.md
Comment (RushFTK) [inline wiki/Ranking_Criteria/zh.md:17]:
````suggestion
- 工具不能代替阅读此页面或人工检查的必要性。
````

File: wiki/Ranking_Criteria/zh.md
Comment (RushFTK) [inline wiki/Ranking_Criteria/zh.md:19]:
````suggestion
- 工具是用于帮助你检查谱面，而不是代替你检查谱面。
````

File: wiki/Ranking_Criteria/zh.md
Comment (RushFTK) [reply wiki/Ranking_Criteria/zh.md:17]: "阅读此页"与词语"操作"不匹配。

File: wiki/Ranking_Criteria/zh.md
Comment (RushFTK) [reply wiki/Ranking_Criteria/zh.md:19]: 原文含义毕竟是促使玩家自己审一遍铺面，主要强调谱师的主观用法而不是AiMod的性能。能其实还是能的（）

**Review by RushFTK — CHANGES_REQUESTED** [review]
No problem @Walavouchey , here is the review as requested.
in my opinion, the second review is necessary to change, while others are opinion.

**Review by RushFTK — APPROVED** [review]
Looks fine for native speakers now.

**Comment by Walavouchey** [comment]
it looks like all the rc articles have the same issue as in https://github.com/ppy/osu-wiki/pull/6851#discussion_r808927887. i'll update all of them in a separate pr

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**
